### PR TITLE
Update filenames

### PR DESCRIPTION
**Pull request by rannick**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/rnafusion _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +41,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `Arriba` visualisation now runs for FusionInspector (combined tools) results, not only `Arriba` results
 - Updated metro map with trimming options and placed `Arriba` visualisation after `FusionInspector`
 - Exit with error when using squid in combination with any ensembl version different from 102
+- Renaming [#395](https://github.com/nf-core/rnafusion/pull/395)
```
Comment (fevac) [inline CHANGELOG.md:44]:
```suggestion
- Renamed output files [#395](https://github.com/nf-core/rnafusion/pull/395)
```

File: conf/modules.config [hunk 2]
```diff
@@ -229,6 +213,7 @@ process {
 
     withName: SAMTOOLS_VIEW_FOR_ARRIBA {
         ext.args = { "--output-fmt cram" }
+        ext.prefix = { "${meta.id}_star_for_arriba" }
```
Comment (fevac) [inline conf/modules.config:216]: Out of the scope of this PR, but I wonder if we would want more general names (e.g. `alignment_for_arriba`) at some point. Not sure really

**Review by fevac — CHANGES_REQUESTED** [review]
Nice! 🌟  can we test with the minimum dataset?

File: conf/modules.config (hunk 2 above)
Comment (rannick) [reply conf/modules.config:216]: I don't think we should change the name of the process, with how it is currently it is easy to understand which process was executed. We can change the name of the cram file to something else for sure

Comment (fevac) [reply conf/modules.config:216]: let's leave it as it is for now :)

**Review by fevac — APPROVED** [review]
🚀 💯

**Pull request merged**